packages/css/src/components/text-area/text-area.scss

```diff
@@ -8,14 +8,15 @@
 @mixin reset-textarea {
   -webkit-appearance: none; // Reset appearance for Safari < 15.4
   appearance: none; // Reset native appearance, this causes issues on iOS and Android devices
-  border: none;
   border-radius: 0; // Reset rounded borders on iOS devices
   margin-block: 0;
 }
 
 .ams-text-area {
   background-color: var(--ams-text-area-background-color);
-  box-shadow: var(--ams-text-area-box-shadow);
+  border-color: var(--ams-text-area-border-color);
+  border-style: var(--ams-text-area-border-style);
+  border-width: var(--ams-text-area-border-width);
```

Comment:
Wrong line, but: a disabled text area still gets a box shadow on hover

Reply:
Ah sorry, I seem to have forgotten that one in the last commit. Added now.